indexer/services/l2/service.go

```diff
@@ -356,8 +330,16 @@ func (s *Service) GetIndexerStatus(w http.ResponseWriter, r *http.Request) {
 	server.RespondWithJSON(w, http.StatusOK, status)
 }
 
-func (s *Service) GetWithdrawalStatus(w http.ResponseWriter, r *http.Request) {
-	// Temporary stub until rest of indexer is landed
+func (s *Service) GetWithdrawalBatch(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
```

Comment:
I feel like we should assign `hash := vars["hash"]` and then check to make sure its not empty?

Reply:
Sure I can do this.

Reply:
Done.